doc/source/whatsnew/v0.23.1.txt

```diff
@@ -59,7 +59,7 @@ Conversion
 Indexing
 ^^^^^^^^
 
--
+- Bug in :meth:`Series.reset_index` where appropriate error was not raised with a non-named level (:issue:`20925`)
```

Comment:
I don't get the "a non-named level". Wouldn't it be "an invalid level name"?

Reply:
Agreed. Done.